CONTRIBUTORS.yaml

```diff
@@ -2435,3 +2435,13 @@ ThibaudGlinez:
     affiliations:
       - pndb
       - elixir-europe
+
+
+tbrown91:
+    name: Tom Brown
+    email: brown@izw-berlin.de
+    orcid: 0000-0001-8293-4816
+    joined: 2024-08
+    affiliations:
+      - IZW
```

Comment:
Those are new to GTN and need to be defined in the https://github.com/galaxyproject/training-material/blob/main/CONTRIBUTORS.yaml

The linter complains with:

```
CONTRIBUTORS.yaml  has errors
  [/tbrown91/affiliations/0] 'IZW': invalid enum value.
  [/tbrown91/affiliations/1] 'ERGA': invalid enum value.
```

Reply:
I'm happy for now not having an affiliation - I'm not really sure I should have ERGA in the FUNDERS list